roles/os_hardening/tasks/user_accounts.yml

```
@@ -45,3 +45,21 @@
     createhome: false
   with_flattened:
     - '{{ sys_accs_cond | default([]) | difference(os_ignore_users) | list }}'
+
+- name: get all home directories in /home, but skip ignored users
+  find:
+    paths: /home/
+    recurse: false
+    file_type: directory
+    excludes: "{{ os_ignore_home_folder_users | join(',') }}"
+  register: home_directories
+  when: os_chmod_home_folders | bool
+
+- name: set ownership of /home directories to 0700
+  file:
+    mode: 0700
+    path: "{{ item }}"
+    state: directory
+  loop:
+    - "{{ home_directories.files | map(attribute='path') | list }}"
+  when: os_chmod_home_folders | bool
```

```suggestion
- name: set ownership of /home directories to 0700
  file:
    mode: 0700
    path: "{{ item.path }}"
    state: directory
  loop:
    - "{{ home_directories.files }}"
  when: os_chmod_home_folders | bool
```

I'd like to use the same approach here, as we used with cron files, that makes it more consistent

https://github.com/dev-sec/ansible-collection-hardening/blob/36bc71fe516f8fdf1c1576e7b1311ae4c576dade/roles/os_hardening/tasks/cron.yml#L22-L28